src/main/webapp/i18n/de/programmingExercise.json

```
@@ -712,6 +712,18 @@
                 "pushExercise": "Lade den Aufgaben-Code in das entsprechende Repository. Es wird in <code>{{checkoutDirectory}}</code> ausgecheckt.",
                 "pushSolution": "Lade den Lösungs-Code in das entsprechende Repository. Es wird nicht ausgecheckt.",
                 "pushTest": "Lade den Test-Code in das entsprechende Repository. Es wird in <code>{{checkoutDirectory}}</code> ausgecheckt."
+            },
+            "dockerFlags": {
+                "disableNetworkAccess": {
+                    "title": "Internetzugang beim Ausführen der Studierendeneinreichung deaktivieren",
```

Not quite sure where it is placed in the UI, would a more active language be fitting into the UI?
```suggestion
                    "title": "Deaktiviere den Internetzugang beim beim Ausführen der Studierendeneinreichung",
```